RELEASES.md

```diff
@@ -1,3 +1,100 @@
+Version 1.82.0 (2024-10-17)
+==========================
+
+<a id="1.82.0-Language"></a>
+
+Language
+--------
+- [Don't make statement nonterminals match pattern nonterminals](https://github.com/rust-lang/rust/pull/120221/)
+- [Implement lint against ambiguous negative literals](https://github.com/rust-lang/rust/pull/121364/)
```

Comment:
While this lint was implemented, it was revert to allow-by-default over concern about usability.

@Mark-Simulacrum What should be done about thing that are no longer release notes worthy? Should we close the release note tracking issue?

Reply:
Yes, that should be sufficient to remove the item from this listing (once I rerun the tool). Essentially all the edits proposed here will need to be transferred elsewhere and then ideally are self-resolved by the update.

Reply:
Great, to hear.

Btw, I actually can't find the relnote tracking issue concerning this item #121364, was it pull from one or is there still a fallback from the `finished-final-comment-period` labels.